src/irmin-pack/unix/dispatcher.ml

```diff
@@ -250,4 +250,80 @@ module Make (Fm : File_manager.S with module Io = Io.Unix) :
   let shrink_accessor_exn a ~new_len =
     if new_len > a.len then failwith "shrink_accessor_exn to larger accessor";
     { a with len = new_len }
+
+  let create_sequential_accessor_exn location rem_len ~poff ~len =
+    if len > rem_len then raise (Errors.Pack_error `Read_out_of_bounds)
+    else { poff; len; location }
+
+  let create_sequential_accessor_from_range_exn location rem_len ~poff ~min_len
+      ~max_len =
+    let len =
+      if rem_len < min_len then raise (Errors.Pack_error `Read_out_of_bounds)
+      else if rem_len > max_len then max_len
+      else rem_len
+    in
+    { poff; len; location }
+
+  let create_sequential_accessor_seq t ~min_header_len ~max_header_len ~read_len
+      =
+    let chunks =
+      match Fm.mapping t.fm with
+      | Some mapping ->
+          let chunks = ref [] in
```

Comment:
Perhaps we name this variable something else in light of upcoming suffix chunks?

Reply:
Will prefix_blocks or even prefix_chunks do the trick ?